acceptance-test-jest-config.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
i think you can have much smaller config, as it only needs a config for one type of test with a specific setup.
still, it seems to work

Reply:
Addressing